templates/pages/service.json

```
@@ -0,0 +1,50 @@
+{
+  "name": "Deployment",
```

Do we want to name it _Deployment_ or perhaps

```suggestion
  "name": "Services",
```

e.g. there is "Local Blueprints" (plural)